lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -146,7 +156,7 @@ getBlockHeader = label "getBlockHeader" $
         -- We could make sure we get the right kind of proof, but we don't need to.
         -- Just checking that the length is not totally wrong, is much simpler
         -- and gives us sanity about the binary format being correct.
-        read' <- fromIntegral <$> bytesRead
+        read' <- fromEnum <$> bytesRead
```

Comment:
I think we can safely use `fromIntegral :: Int64 -> Word16` here. Because within a block of `isolate size`, the return value of `bytesRead` is always `<= size`.